src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -136,9 +136,9 @@ Set<String> tabulate() throws TabulationCancelledException {
     // - If winnerElectionMode is "Bottoms-up using percentage threshold", we loop until all
     // remaining candidates have vote shares that meet or exceed that threshold.
     //
-    // At each iteration, we'll either a) identify one or more
+    // At each iteration, we'll either a. identify one or more
     // winners and transfer their votes to the remaining candidates (if we still need to find more
-    // winners), or b) eliminate one or more candidates and gradually transfer votes to the
+    // winners), or b. eliminate one or more candidates and gradually transfer votes to the
```

Comment:
Super nit: I'd revert these two back to "a)" and "b)".

Reply:
Ack; prefer to leave this way because grammar checking plugins in intellij consider these to be unclosed parentheses.

Reply:
Got it, sounds good then. Funny it doesn't pick up on uncapitalized letters after periods, haha.